lib/PicoDb/Table.php

```diff
@@ -728,14 +728,14 @@ public function offset($value)
     }
 
     /**
-     * Group by
+     * Group By
      *
-     * @access public
+     * @param ...$columns
```

Comment:
You can add a type for variadic arguments that IDEs should understand.

```
@param string ...$columns
```

Reply:
there is probably a few others that are variadic just like this one too.